assignment.js

```diff
@@ -113,6 +124,50 @@ console.log('Problem 3:')
 
 // Add your code below this line
 
+
+let playerAChoice = 'scissors'
+let playerBChoice = 'paper'
+let winner = whoWins(playerAChoice, playerBChoice)
+
+//for the user's info
+console.log(`Player A chose ${playerAChoice}`)
+console.log(`Player B chose ${playerBChoice}`)
+
+//if both players choose the same item it's a tie. Otherwise, evaluate who wins and print it to console.
+playerAChoice === playerBChoice ? console.log('It\'s a tie!') : console.log(`${winner} wins!`)
+
+//this function will evaluate and return a winner (player A or B)
+function whoWins(playerA, playerB) {
+
+  //determine who wins based on playerA's choice
+  switch (playerA) {
+    case 'rock':
+      if (playerB === 'scissors') {
+        return 'Player A'
+      } else if (playerB === 'paper') {
+        return 'Player B'
+      }
+      break
+
+    case 'paper':
+      if (playerB === 'rock') {
+        return 'Player A'
+      } else if (playerB === 'scissors') {
+        return 'Player B'
+      }
+      break
+
+    case 'scissors':
+      if (playerB === 'paper') {
+        return 'Player A'
+      } else if (playerB === 'rock') {
+        return 'Player B'
+      }
+      break
+    default:
+      console.log('Both players must choose an item!')
+  }
+}
 // Add your code above this line
 
 console.log('')
```

Comment:
@sad3xoxo Great Job on this solution. Good job creating the function to determine the winner of the game. That's an effective way to use the switch and conditional statements together to check for the winning outcomes.

Take a look at this solution [here]()